common/conf/reference.conf

```diff
@@ -2,58 +2,87 @@
 # https://github.com/typesafehub/config
 # ~~~~~
 
+sunrise {
```

Comment:
This is important: I've put everything inside "sunrise". It makes sense to have all our configuration inside "sunrise", also to avoid name collision. Play also has everything under "play".